### DISCUSSION TITLE
How to remove shadow from button？

### DISCUSSION THREAD

**Discussion by CeSun** [AvaloniaCommunity/Material.Avalonia#188] [Q&A] [answered]
I want to use Button as bottom navbar, but don't want the shadow it brings.
![90H7YA%_X~4GDOQFGT%F}RA](https://user-images.githubusercontent.com/54771283/197335466-626c4ffb-bee6-48b3-89f9-407cb5098a87.png)

```xaml
<UserControl xmlns="https://github.com/avaloniaui"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
             xmlns:viewModels="clr-namespace:Forum.ViewModels"
             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
             x:Class="Forum.Views.MainView"
             Foreground="Black"
             Background="#F1F1F1">
  <Design.DataContext>
    <viewModels:MainViewModel />
  </Design.DataContext>
	<Panel>
		<StackPanel VerticalAlignment="Top" >
			<Border BoxShadow="0 0 10 2 Gray">
				<Panel  Height="70"  Background="BlueViolet">
					<Label VerticalAlignment="Center" Foreground="#fff" FontWeight="Bold" Margin="20 0 0 0" FontSize="30" Content="知乎"></Label>
				</Panel >
			</Border>
			<Border Padding="0 10 0 20">
			<ItemsRepeater Items="{Binding MyList}" >
				<ItemsRepeater.ItemTemplate>
					<DataTemplate>
						<Button  Margin="0 0 0 5" Height="40" Width="100" Padding="0">
							<Label FontSize="18" Content="{Binding}">
							</Label>
						</Button>
					</DataTemplate>
				</ItemsRepeater.ItemTemplate>
			</ItemsRepeater >
			</Border>
		</StackPanel>
		<Grid ColumnDefinitions="1*,1*,1*,1*" VerticalAlignment="Bottom" Height="70" Background="White">
			<Panel Grid.Column="0">
				<Label FontSize="18" Foreground="#333" VerticalAlignment="Bottom" HorizontalContentAlignment="Center" Content="Home"></Label>
				<Button CornerRadius="0" BorderBrush="Transparent" Background="Transparent"></Button>
			</Panel>
			<Panel Grid.Column="1">
				<Label FontSize="18"  Foreground="#333" VerticalAlignment="Bottom" HorizontalContentAlignment="Center" Content="Category"></Label>
				<Button CornerRadius="0" BorderBrush="Transparent" Background="Transparent"></Button>
			</Panel>
			<Panel Grid.Column="2" >
				<Label FontSize="18" Foreground="#333" VerticalAlignment="Bottom" HorizontalContentAlignment="Center" Content="Message"></Label>
				<Button CornerRadius="0" BorderBrush="Transparent" Background="Transparent"></Button>
			</Panel>
			<Panel Grid.Column="3">
				<Label FontSize="18" Foreground="#333" VerticalAlignment="Bottom" HorizontalContentAlignment="Center" Content="Profiles"></Label>
				<Button CornerRadius="0" BorderBrush="Transparent" Background="Transparent"></Button>
			</Panel>
		</Grid>

	</Panel>
</UserControl>

```

**Comment by SKProCH** [comment, marked as answer]
Hello, you can set `ShadowAssist.Depth` to control shadows

**Reply by CeSun** [reply]
![image](https://user-images.githubusercontent.com/54771283/197356934-eb25cdb0-fdd4-49bd-97db-e76b8ac9dbf4.png)
Like this? But this compilation error.

I also tried to add the namespace Material.Styles.Assists, but the ide indicated that there was no such namespace

**Reply by SKProCH** [reply]
You need to add namespace `xmlns:assists="clr-namespace:Material.Styles.Assists;assembly=Material.Styles"` and use `assists:ShadowAssist.ShadowDepth="Depth0"`. 
![image](https://user-images.githubusercontent.com/29896317/197391070-944c3337-d21b-4417-8bb1-7bf7b3f8647d.png)
![image](https://user-images.githubusercontent.com/29896317/197391080-1e375f1a-3ff8-489d-8326-b9154dd688a7.png)


Also, consider take a look at JetBrains Rider, it help a lot with developing with Avalonia.

**Reply by CeSun** [reply]
> You need to add namespace `xmlns:assists="clr-namespace:Material.Styles.Assists;assembly=Material.Styles"` and use `assists:ShadowAssist.ShadowDepth="Depth0"`. ![image](https://user-images.githubusercontent.com/29896317/197391070-944c3337-d21b-4417-8bb1-7bf7b3f8647d.png) ![image](https://user-images.githubusercontent.com/29896317/197391080-1e375f1a-3ff8-489d-8326-b9154dd688a7.png)
> 
> Also, consider take a look at JetBrains Rider, it help a lot with developing with Avalonia.

It worked, thanks a lot!
